packages/datetime/src/common/_react-day-picker-overrides.scss

```diff
@@ -0,0 +1,27 @@
+// Copyright 2023 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0.
+
+@import "@blueprintjs/colors/lib/scss/colors";
+@import "@blueprintjs/core/src/common/variables";
+
+// omit file extension, otherwise this will emit a `@import url()` rule
+@import "react-day-picker-8/dist/style";
```

Comment:
For files ported from `@blueprintjs/datetime2` we use the `react-day-picker-8` alias:
https://github.com/palantir/blueprint/blob/ca86eebdf9e7ed242d0cfd8abaf73ac376c9a5f1/packages/datetime2/src/common/_react-day-picker-overrides.scss#L8